chromium_src/components/update_client/update_checker.h

```diff
@@ -0,0 +1,95 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_UPDATE_CLIENT_UPDATE_CHECKER_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_UPDATE_CLIENT_UPDATE_CHECKER_H_
+
+#include "../../../../components/update_client/update_checker.h"
+
+#include "components/update_client/configurator.h"
```

Comment:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Implemented in d43fe7e.